docs/pages/react-native/getting-started/getting-started-expo.mdx

```diff
@@ -220,6 +236,61 @@ npx expo prebuild --platform ios
 
 </CodeBlocks>
 
+## Run the app
+
+Because the app is using native modules, you cannot run it with expo go and instead need to use development builds. You can do this with the `android` and `ios` commands:
+
+<CodeBlocks>
+
+```bash npm(android)
+npm run android
+```
+
+```bash yarn(android)
+yarn android
+```
+
+```bash npm(ios)
+npm run ios
+```
+
+```bash yarn(ios)
+yarn ios
+```
+
+</CodeBlocks>
+
+## Common Issues
```

Comment:
Maybe this isn't an issue with yarn or other package manager but when using npm, I had to add this to my package.json to force all the packages to use the same react version

```
  "overrides": {
    "react": "19.0.0"
  }
  ```

Reply:
oh gotcha, yea I got around that by using `npm install --save --legacy-peer-deps` 

I can add this there. Is react the only one? or do we need to do the same for react-dom and react-native?

Reply:
ahh I see, react was the only one that caused me issues